var/spack/repos/builtin/packages/xpmem/package.py

```diff
@@ -58,18 +58,26 @@ class Xpmem(AutotoolsPackage):
 
     # It will become possible to disable the kernel module only starting 2.6.6:
     # https://github.com/hjelmn/xpmem/pull/24
-    conflicts("~kernel-module", when="@:2.6.5")
+    conflicts(
```

Comment:
Can you turn this into a conditional variant instead?

variant("...", when="...")

Reply:
@haampie stop using conditional variants, please :) or make `spack install xpmem@:2.6.5+kernel-module` work.